scripts/submit_skims.sh

```diff
@@ -92,6 +92,149 @@ while [[ $# -gt 0 ]]; do
+	["GluGluToRadionToHHTo2B2Tau_M-400_"]="Rad400"
+	["GluGluToRadionToHHTo2B2Tau_M-450_"]="Rad450"
+	["GluGluToRadionToHHTo2B2Tau_M-500_"]="Rad500"
+	["GluGluToRadionToHHTo2B2Tau_M-550_"]="Rad550"
+	["GluGluToRadionToHHTo2B2Tau_M-600_"]="Rad600"
+	["GluGluToRadionToHHTo2B2Tau_M-650_"]="Rad650"
+	["GluGluToRadionToHHTo2B2Tau_M-700_"]="Rad700"
+	["GluGluToRadionToHHTo2B2Tau_M-750_"]="Rad750"
+	["GluGluToRadionToHHTo2B2Tau_M-800_"]="Rad800"
+	["GluGluToRadionToHHTo2B2Tau_M-850_"]="Rad850"
+	["GluGluToRadionToHHTo2B2Tau_M-900_"]="Rad900"
+	["GluGluToRadionToHHTo2B2Tau_M-1000_"]="Rad1000"
+	["GluGluToRadionToHHTo2B2Tau_M-1250_"]="Rad1250"
+	["GluGluToRadionToHHTo2B2Tau_M-1500_"]="Rad1500"
+	["GluGluToRadionToHHTo2B2Tau_M-1750_"]="Rad1750"
+	["GluGluToRadionToHHTo2B2Tau_M-2000_"]="Rad2000"
+	["GluGluToRadionToHHTo2B2Tau_M-2500_"]="Rad2500"
+	["GluGluToRadionToHHTo2B2Tau_M-3000_"]="Rad3000"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-250_"]="Grav250"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-260_"]="Grav260"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-270_"]="Grav270"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-280_"]="Grav280"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-300_"]="Grav300"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-320_"]="Grav320"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-350_"]="Grav350"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-400_"]="Grav400"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-450_"]="Grav450"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-500_"]="Grav500"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-550_"]="Grav550"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-600_"]="Grav600"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-650_"]="Grav650"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-700_"]="Grav700"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-750_"]="Grav750"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-800_"]="Grav800"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-850_"]="Grav850"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-900_"]="Grav900"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-1000_"]="Grav1000"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-1250_"]="Grav1250"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-1500_"]="Grav1500"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-1750_"]="Grav1750"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-2000_"]="Grav2000"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-2500_"]="Grav2500"
+	["GluGluToBulkGravitonToHHTo2B2Tau_M-3000_"]="Grav3000"
+
+	# Background
+    ["TTToHadronic"]="TT_Hadronic"
+    ["TTTo2L2Nu"]="TT_FullyLep"
+    ["TTToSemiLeptonic"]="TT_SemiLep"
+
+	["DYJets.+_M-50_T.+amc"]="DY_Incl"
+    ["DYJetsToLL_LHEFilterPtZ-0To50"]="DY_PtZ0To50"
+    ["DYJetsToLL_LHEFilterPtZ-50To100"]="DY_PtZ50To100"
+    ["DYJetsToLL_LHEFilterPtZ-100To250"]="DY_PtZ100To250"
+    ["DYJetsToLL_LHEFilterPtZ-250To400"]="DY_PtZ250To400" 
+    ["DYJetsToLL_LHEFilterPtZ-400To650"]="DY_PtZ400To650"
+    ["DYJetsToLL_LHEFilterPtZ-650ToInf"]="DY_PtZ650ToInf"
+    ["DYJetsToLL_0J"]="DY_0J"
+    ["DYJetsToLL_1J"]="DY_1J"
+    ["DYJetsToLL_2J"]="DY_2J"
+
+    ["WJetsToLNu_T.+madgraph"]="WJest_Incl" # for 0 < HT < 70
```

Comment:
`WJest` -> `WJets` 

Also not a strong opinion, but since we only use this for 0<HT<70, maybe it could be labeled the same as the sliced samples? (ie `WJets_HT0To70`)